bambam.py

```diff
@@ -23,247 +23,244 @@
-
-        # mouse button up
-        elif event.type == MOUSEBUTTONUP:
-            mouse_down = False
-
-    return quit_pos
-
-
-# Prints an image at a random location
-def print_image():
-    #global swidth, sheigh
-    img = images[random.randint(0, len(images) - 1)]
-    w = random.randint(0, swidth  - img.get_width())
-    h = random.randint(0, sheight - img.get_height())
-    screen.blit(img, (w, h))
-
-# Is the key that was pressed alphanumeric
-def is_latin(key):
-    return key < 255 and (chr(key) in string.ascii_letters or chr(key) in string.digits)
-
-# Prints a letter at a random location
-def print_letter(char):
-    global args
-    font = pygame.font.Font(None, 256)
-    if args.uppercase:
-        char = char.upper()
-    text = font.render(char, 1, colors[random.randint(0, len(colors) - 1)])
-    textpos = text.get_rect()
-    center = (textpos.width // 2, textpos.height // 2)
-    w = random.randint(0 + center[0], swidth - center[0])
-    h = random.randint(0 + center[1], sheight - center[1])
-    textpos.centerx = w
-    textpos.centery = h
-    screen.blit(text, textpos)
-
-def glob_data(pattern):
-    global progInstallBase
-    return glob.glob(os.path.join(progInstallBase, 'data', pattern))
-
-# Global variables
-#
-progInstallBase = None
-mouse_down = None
-screen = None
-sequence = None
-sound_muted = None
-args = None
-sounds = None
-colors = None
-images = None
-swidth = None
-sheight = None
-background = None
-
-# Main application
-#
-def main():
-    global progInstallBase, mouse_down, screen, sequence, sound_muted, args, sounds, colors, images, swidth, sheight, background
-    progInstallBase = os.path.dirname(os.path.realpath(sys.argv[0]));
+            # mouse button up
+            elif event.type == MOUSEBUTTONUP:
+                self.mouse_down = False
+
+        return quit_pos
+
+
+    def print_image(self):
+        ''' Prints an image at a random location '''
+        img = self.images[random.randint(0, len(self.images) - 1)]
+        w = random.randint(0, self.swidth  - img.get_width())
+        h = random.randint(0, self.sheight - img.get_height())
+        self.screen.blit(img, (w, h))
+
+    def print_letter(self, char):
+        ''' Prints a letter at a random location '''
+        font = pygame.font.Font(None, 256)
+        if self.args.uppercase:
+            char = char.upper()
+        text = font.render(char, 1, self.colors[random.randint(0, len(self.colors) - 1)])
+        textpos = text.get_rect()
+        center = (textpos.width // 2, textpos.height // 2)
+        w = random.randint(0 + center[0], self.swidth - center[0])
+        h = random.randint(0 + center[1], self.sheight - center[1])
+        textpos.centerx = w
+        textpos.centery = h
+        self.screen.blit(text, textpos)
+
+    def glob_data(self, pattern):
+        return glob.glob(os.path.join(self.progInstallBase, 'data', pattern))
+
+    def run(self):
+        ''' Main application loop '''
```

Comment:
Well, given that the loop is just 3 lines at the end of this function, I think a better docstring would be something like `Main application entry point.`?